src/web/components/ApiKeyManagement/ApiRolesCell.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import ApiRolesCell from './ApiRolesCell';
+
+const meta: Meta<typeof ApiRolesCell> = {
+  component: ApiRolesCell,
+  title: 'ApiManagement/Api Roles cell',
+  decorators: [
+    (Story) => (
+      <div>
+        <table>
+          <thead>
+            <tr>
+              <th>Roles</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Story />
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    ),
+  ],
+};
+export default meta;
+
+type Story = StoryObj<typeof ApiRolesCell>;
+
+export const ManyRoleShowingRoleName: Story = {
+  args: {
+    apiRoles: [
+      { id: 1, roleName: 'Role1', externalName: 'Role 1' },
```

Comment:
We've seen a lot of confusion in sprint showcases and other feedback sessions before when we had fake names - I think it's worthwhile to make these match the actual values we use.

Reply:
I think the worst was when we had some stories showing participant types as "Type 1", "Type 2", etc. - people kept losing track of what they actually were, and it got much easier once we switched to using "Advertiser", "DSP", etc.

Reply:
Yeah that makes sense